tests/NgTests/RegistrationCollectorTests.cs

```diff
@@ -306,11 +306,44 @@ public static IEnumerable<object[]> PageContent
             await _target.Run(front, back, CancellationToken.None);
 
             // Assert
-            // Verify the contents of the SemVer 1.0.0 storage
+            var legacyCursor = _legacyStorage.Content.FirstOrDefault(pair => pair.Key.PathAndQuery.EndsWith("cursor.json"));
+            Assert.NotNull(legacyCursor.Key);
+            var legacyIndex = _legacyStorage.Content.FirstOrDefault(pair => pair.Key.PathAndQuery.EndsWith("/testpackage.semver2/index.json"));
```

Comment:
Yup, good idea. This is already implied by the cursor `NotNull` assertion and the count equaling 1 assertion. But it's best to be explicit here.